template.tpl

```diff
@@ -73,49 +118,89 @@ ___TEMPLATE_PARAMETERS___
     "groupStyle": "ZIPPY_CLOSED",
     "subParams": [
       {
-        "type": "TEXT",
-        "name": "scriptUrlPattern",
-        "displayName": "Script Url Pattern",
-        "simpleValueType": true,
-        "help": "Pattern of the JS agent script URL."
-      },
-      {
-        "type": "TEXT",
-        "name": "endpoint",
-        "displayName": "Endpoint",
-        "simpleValueType": true,
-        "help": "Server API URL. Should be only used with Subdomain integration."
-      },
-      {
-        "type": "TEXT",
-        "name": "tag",
-        "displayName": "tag",
-        "simpleValueType": true,
-        "help": "A customer-provided value or an object that will be saved together with the identification event"
-      },
-      {
-        "type": "TEXT",
-        "name": "linkedId",
-        "displayName": "linkedId",
-        "simpleValueType": true,
-        "help": "A way of linking current identification event with a custom identifier"
-      },
-      {
-        "type": "CHECKBOX",
-        "name": "extendedResult",
-        "checkboxText": "Extended result",
-        "simpleValueType": true
+        "type": "GROUP",
+        "name": "Agent settings",
+        "displayName": "",
+        "groupStyle": "NO_ZIPPY",
+        "subParams": [
+          {
+            "type": "TEXT",
+            "name": "scriptUrlPattern",
+            "displayName": "Script Url Pattern",
+            "simpleValueType": true,
+            "help": "Pattern of the JS agent script URL."
+          },
+          {
+            "type": "TEXT",
+            "name": "endpoint",
+            "displayName": "Endpoint",
+            "simpleValueType": true,
+            "help": "Server API URL. Should be only used with Subdomain integration."
```

Comment:
- not the `Subdomain integration` - it's a Custom subdomain setup.
- and proxy integration?